bootloader/bootloader.go

```diff
@@ -226,6 +226,15 @@ type TrustedAssetsBootloader interface {
 	BootChain(runBl Bootloader, kernelPath string) ([]BootFile, error)
 }
 
+// RebootArgumentsBootloader needs arguments to the reboot syscall when snaps
+// are being updated.
+type RebootArgumentsBootloader interface {
+	Bootloader
+
+	// GetRebootArguments returns the needed reboot arguments
+	GetRebootArguments() string
```

Comment:
I am highly suspicious that this will not end up needing to be able to return an error